tests/exception/bulkwriteexception-getwriteresult-001.phpt

```diff
@@ -1,22 +1,60 @@
 --TEST--
 MongoDB\Driver\Exception\BulkWriteException::getWriteResult()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
 --FILE--
 <?php
+require_once __DIR__ . "/../utils/basic.inc";
 
-$exception = new MongoDB\Driver\Exception\BulkWriteException();
-$writeResult = ['x' => 1];
+$manager = create_test_manager();
 
-$reflection = new ReflectionClass($exception);
-
-$resultDocumentProperty = $reflection->getProperty('writeResult');
-$resultDocumentProperty->setAccessible(true);
-$resultDocumentProperty->setValue($exception, $writeResult);
-
-var_dump($writeResult === $exception->getWriteResult());
+try {
+    $bulk = new MongoDB\Driver\BulkWrite;
+    $bulk->insert(['_id' => 1]);
+    $bulk->insert(['_id' => 1]);
+    $result = $manager->executeBulkWrite(NS, $bulk);
+} catch (MongoDB\Driver\Exception\BulkWriteException $e) {
+    var_dump($e->getWriteResult());
+}
 
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
-bool(true)
+--EXPECTF--
+object(MongoDB\Driver\WriteResult)#%d (9) {
```

Comment:
Note that we typically use `%d` for the number of object properties as well. That keeps the diff smaller if we end up adding additional properties down the line.

Change can be made here and in the other two `object(` lines below.

Reply:
Good point, done.